framework/meta/src/cli_args/cli_args_contract.rs

```diff
@@ -65,7 +65,13 @@ pub enum ContractCliAction {
         name = "snippets",
         about = "Generates a snippets project, based on the contract ABI."
     )]
-    GenerateSnippets(GenerateSnippetsArgs),
+    GenerateSnippets(GenerateOverwriteArg),
+
+    #[command(
+        name = "proxy",
+        about = "Generates a proxy, based on the contract ABI."
+    )]
+    GenerateProxies(GenerateOverwriteArg),
```

Comment:
Don't reuse the same arg type for both snippets and proxies. Even if, by coincidence they currently contain the same fields, they are likely to eveolve in different directions in the future.